src/styles/global.css

```
@@ -175,4 +175,17 @@ img:not([src]) {
   top: 0;
+}
+
+/* Screen Reader Only Utility */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)